acis_thermal_check/state_builder.py

```diff
@@ -143,7 +143,7 @@ def get_prediction_states(self, tbegin):
 
         # Get timeline load segments including state0 and beyond.
         timeline_loads = self.db.fetchall("""SELECT * from timeline_loads
-                                          WHERE datestop > '%s'
+                                          WHERE datestop >= '%s'
```

Comment:
Ping @Gregg140 -- this one change fixes the differences we saw in states.dat between some loads between the two different state builders.

Reply:
That looks good.